srcpkgs/fwupd/template

```diff
@@ -1,23 +1,18 @@
 # Template file for 'fwupd'
 pkgname=fwupd
-version=1.5.8
+version=1.6.3
 revision=1
 build_style=meson
 build_helper="gir"
 # tests require unpackaged umockdev
-configure_args="-Dsupported_build=true
- -Dconsolekit=false -Dgtkdoc=true -Dintrospection=true
- -Dsystemd=false -Dplugin_altos=false -Dtests=false
- -Dpkcs7=false -Db_lto=false -Defi-includedir=${XBPS_CROSS_BASE}/usr/include/efi
- -Defi-ldsdir=${XBPS_CROSS_BASE}/usr/lib -Defi-ld=${XBPS_CROSS_BASE}/bin/ld
- -Defi-objcopy=${XBPS_CROSS_BASE}/bin/objcopy
- -Defi-libdir=${XBPS_CROSS_BASE}/usr/lib -Delogind=true"
-hostmakedepends="dejavu-fonts-ttf gnutls-tools help2man pkg-config gcab
- vala glib-devel polkit gettext gtk-doc pango python3-gobject python3-Pillow"
+configure_args="-Dsupported_build=true -Dconsolekit=false -Dintrospection=true
+ -Dsystemd=false -Dplugin_altos=false -Dtests=false -Delogind=true"
+hostmakedepends="dejavu-fonts-ttf gnutls-tools pkg-config gcab gi-docgen
+ vala glib-devel polkit gettext pango python3-gobject python3-Pillow"
 makedepends="libxmlb-devel cairo-devel colord-devel libarchive-devel
  gnutls-devel gpgme-devel json-glib-devel libgusb-devel polkit-devel
  sqlite-devel libsoup-devel gcab-devel pango-devel elogind-devel
- tpm2-tss-devel libjcat-devel libcurl-devel"
+ tpm2-tss-devel libjcat-devel libcurl-devel fwupd-efi"
 depends="udisks2"
 conf_files="
```

Comment:
Does `fwupd-efi` get bundled into `fwupd`? Otherwise I think we should add it to `depends` as well. (I'm pretty sure we should, seeing as the `/usr/libexec/fwupd/efi/fwupdx64.efi` file used to be in `fwupd`.